tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -43,14 +43,41 @@ type UserLog = Array<{ type: keyof typeof logger; message: string }>;
 export default async function (tree: Tree, schema: MigrateConvergedPkgGeneratorSchema) {
   const userLog: UserLog = [];
 
+  validateUserInput(tree, schema);
+
   if (schema.stats) {
     printStats(tree, schema);
     // eslint-disable-next-line @typescript-eslint/no-empty-function
     return () => {};
   }
 
-  validateUserInput(tree, schema);
+  if (schema.all) {
+    runBatchMigration(tree, userLog);
+  } else {
+    runMigrationOnProject(tree, schema, userLog);
+  }
+
+  await formatFiles(tree);
+
+  return () => {
+    printUserLogs(userLog);
+  };
+}
+
+function runBatchMigration(tree: Tree, userLog: UserLog) {
+  const projects = getProjects(tree);
+
+  projects.forEach((project, projectName) => {
+    if (!isPackageConverged(tree, project)) {
+      userLog.push({ type: 'error', message: `${projectName} is not converged package. Skipping migration...` });
+      return;
+    }
 
+    runMigrationOnProject(tree, { name: projectName }, userLog);
+  });
+}
+
+function runMigrationOnProject(tree: Tree, schema: AssertedSchema, userLog: UserLog) {
```

Comment:
encapsulated all codemod/generator logic into re-usable function which is invoked for standard invocation and multiple times when migrating in batch